geometry_msgs/PoseStamped goal
---
bool ret